Preparing training and test sets
Traceback (most recent call last):
  File "/Applications/PyCharm.app/Contents/helpers/pydev/pydev_run_in_console.py", line 52, in run_file
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/Applications/PyCharm.app/Contents/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/Users/michele/Fraud_Detection_LSTM_Thesis/LSTM_classifier.py", line 97, in <module>
    x_train, y_train = create_dataset(train, look_back=look_back)
  File "/Users/michele/Fraud_Detection_LSTM_Thesis/LSTM_classifier.py", line 62, in create_dataset
    series_x, series_y = get_series(transactions, labels, look_back)
  File "/Users/michele/Fraud_Detection_LSTM_Thesis/LSTM_classifier.py", line 41, in get_series
    isFraudolent = labels.iloc[0:look_back]["isFraud"].max()
AttributeError: 'numpy.ndarray' object has no attribute 'iloc'
PyDev console: starting.

Python 3.6.3 (v3.6.3:2c5fed86e0, Oct  3 2017, 00:32:08) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
